src/utilities/indexer/updateAttestationCount.ts

```diff
@@ -0,0 +1,52 @@
+import { type ICType } from '@kiltprotocol/sdk-js';
+
+import { Op } from 'sequelize';
+
+import { CType as CTypeModel } from '../../models/ctype';
+
+import { logger } from '../logger';
+
+import { matchesGenerator } from './queryFromIndexer';
+
+export async function updateAttestationsCount() {
+  const fields = ['cTypeId: id', 'attestationsCreated', 'registrationBlockId'];
+
+  const queryParams = {
+    entity: 'cTypes',
+    alias: 'attestationsCount',
+    fields,
+  };
+
+  interface QueriedAttestationCount {
+    cTypeId: ICType['$id'];
+    attestationsCreated: number;
+    registrationBlockId: string; // Block Ordinal Number, without punctuation
+  }
+  const entitiesGenerator =
+    matchesGenerator<QueriedAttestationCount>(queryParams);
+
+  for await (const entity of entitiesGenerator) {
+    const { cTypeId, attestationsCreated } = entity;
+
+    const cTypeToUpdate = await CTypeModel.findOne({
+      where: {
+        id: {
+          [Op.eq]: cTypeId,
+        },
```

Comment:
```suggestion
        id: cTypeId,
```
Would this work? Is this more straightforward?

Reply:
Yes, that would also work. 

I don't see a benefit from changing this. It seems more strait forward to me to treat all operators the same way.

Reply:
Fair enough for the mix of operators. When there are only equality operators, it doesn’t make sense to have them explicit though.